README.md

```diff
@@ -70,6 +70,7 @@ These are for demonstration purposes only.
 
 - [x] [Linear](./src/searching/linear_search.rs)
 - [x] [Binary](./src/searching/binary_search.rs)
+- [x] [Closest pair of 2D points](./src/searching/closest_points.rs)
```

Comment:
It's not a search algorithm (they search in an array). This is more like geometry or something...

Reply:
Do you want me to create a "geometry" section then ? (and move the file in a "geometry" folder)

Reply:
I noticed convex hull (which is also a geometry algorithm) is in general, do you want me to use general ?

Reply:
Yes, let's move both of them to Geometry

Reply:
Please re-request my review once this change is made